doc/cabal-project.rst

```diff
@@ -194,8 +194,10 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starting with Cabal 2.4, there is now a stanza
-``source-repository-package`` for specifying packages from an external
-version control.
+``source-repository-package`` for specifying packages stored in a remote
+version control system that cabal should consider during package retrieval.
+This allows use of a package in a remote version control system, rather
```

Comment:
the second sentence overlaps with the first one a lot now. Maybe, the second one should just say:

> This simplifies usage of packages not present on Hackage.

?

Reply:
this feels wrong. my usecase is pointing to a modified version of something that is in hackage, but broken with ghc 9.4+.

Reply:
Well, it could say more: not present on Hackage or present but buggy, and the VCS version can contain a fix. But I don't feel too strong about it.